frontend/public/components/resource-quota.jsx

```diff
@@ -35,64 +55,125 @@ const Row = ({obj: rq}) => <div className="row co-resource-list__item">
-        <div className="col-sm-2 hidden-xs">Capacity</div>
-        <div className="col-sm-3 col-xs-3">Used</div>
-        <div className="col-sm-3 col-xs-3">Max</div>
+const QuotaScopes = ({quota}) => {
+  const scopes = _.get(quota, ['spec', 'scopes']);
+  if (!scopes) {
+    return <dd>-</dd>;
+  }
+  const retVal = scopes.map(scope => {
+    const scopeObj = _.get(quotaScopes, scope);
+    return scopeObj ?
+      <dd key={scope}>
+        <div className="co-resource-quota-scope__label">{scopeObj.label}</div>
+        <div className="co-resource-quota-scope__description">{scopeObj.description}</div>
+      </dd>
+      : <dd key={scope} className="co-resource-quota-scope__label">{scope}</dd>;
+  });
+  return retVal;
+};
+
+const showCharts = resourceTypes => {
+  const chartResourceTypes = ['requests.cpu', 'limits.cpu', 'requests.memory', 'limits.memory'];
+  return _.intersection(resourceTypes, chartResourceTypes).length > 0;
+};
+
+const Details = ({obj: rq}) => {
+  const resourceTypes = getResourceTypes(rq);
+  const showChartRow = showCharts(resourceTypes);
+  const popover = (
+    <Popover>
+      Requests are the amount of resources you expect to use. These are used when establishing if the cluster can fulfill your Request.<br />
+      <br />
+      Limits are a maximum amount of a resource you can consume. Applications consuming more than the Limit may be terminated.<br />
+      <br />
+      A cluster administrator can establish limits on both the amount you can Request and your Limits with a &nbsp;
+      <a href={helpLink('dev_guide/compute_resources.html#dev-compute-resources')} target="_blank" rel="noopener">Resource Quota</a>.
+    </Popover>
+  );
+  return <React.Fragment>
+    <div className="co-m-pane__body">
+      <SectionHeading text="Resource Quota Overview" />
+      {showChartRow && <div className="co-resource-quota-chart-row">
+        {_.includes(resourceTypes, 'requests.cpu') &&
+          <div className="co-resource-quota-chart">
+            <Gauge title="CPU Request" percentUsed={getProportions(rq, 'requests.cpu').percentUsed} />
+          </div>}
+        {_.includes(resourceTypes, 'limits.cpu') &&
+        <div className="co-resource-quota-chart">
+          <Gauge title="CPU Limit" percentUsed={getProportions(rq, 'limits.cpu').percentUsed} />
+        </div>}
+        {_.includes(resourceTypes, 'requests.memory') &&
+        <div className="co-resource-quota-chart">
+          <Gauge title="Memory Request" percentUsed={getProportions(rq, 'requests.memory').percentUsed} />
+        </div>}
+        {_.includes(resourceTypes, 'limits.memory') &&
+        <div className="co-resource-quota-chart">
+          <Gauge title="Memory Limit" percentUsed={getProportions(rq, 'limits.memory').percentUsed} />
+        </div>}
+      </div>}
+      <div className="row">
+        <div className="col-sm-6">
+          <ResourceSummary resource={rq} podSelector="spec.podSelector" showNodeSelector={false} />
```

Comment:
I don't think there is a `podSelector` for resource quotas?

Reply:
idk :-) it was in the existing code, and in the mockups, I can remove

Reply:
Let's change it to `showPodSelector={false}`. I can't find anything about pod selectors for quotas. (There is a `scopeSelector`, but that's something else.)